src/components/Dialog/__stories__/Dialog.stories.mdx

```diff
@@ -0,0 +1,506 @@
+              </DialogContentContainer>
+            }
+          >
+            <Button kind={Button.kinds.SECONDARY} onClick={onChangeLeft} active={checkedLeft}>
+              Left
+            </Button>
+          </Dialog>
+        </Flex>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Dialog show triggers
+
+We can choose what will be the related element's trigger which will be responsible for the dialog appearance
+
+<Canvas>
+  <Story name="Show triggers">
+    {() => {
+      // For maintain active state of each button according to the dialog open state (this hooks is available for your usage)
+      const { isChecked: clickButtonActive, onChange: onClickClickButton } = useSwitch({ defaultChecked: false });
+      const { isChecked: hoverButtonActive, onChange: onHoverButton } = useSwitch({ defaultChecked: false });
+      const { isChecked: focusButtonActive, onChange: onFocusButton } = useSwitch({ defaultChecked: false });
+      const modifiers =
+        // for prevent dialog to move while scrolling
+        [
+          {
+            name: "preventOverflow",
+            options: {
+              mainAxis: false
+            }
+          }
+        ];
+      return (
+        <Flex className="monday-storybook-dialog--story-padding" gap={Flex.gaps.MEDIUM}>
+          <Dialog
+            modifiers={modifiers}
+            showTrigger={[Dialog.hideShowTriggers.CLICK]}
+            hideTrigger={[Dialog.hideShowTriggers.CLICK]}
+            content={
+              <DialogContentContainer>
+                <ExampleContent />
+              </DialogContentContainer>
+            }
+          >
+            <Button kind={Button.kinds.SECONDARY} active={clickButtonActive} onClick={onClickClickButton}>
+              On click
+            </Button>
+          </Dialog>
+          <Dialog
+            modifiers={modifiers}
+            showTrigger={[Dialog.hideShowTriggers.MOUSE_ENTER]}
+            hideTrigger={[Dialog.hideShowTriggers.MOUSE_LEAVE]}
+            content={
+              <DialogContentContainer>
+                <ExampleContent />
+              </DialogContentContainer>
+            }
+          >
+            <div onMouseEnter={onHoverButton} onMouseLeave={onHoverButton}>
+              <Button kind={Button.kinds.SECONDARY} active={hoverButtonActive}>
+                On mouse enter
+              </Button>
+            </div>
+          </Dialog>
+          <Dialog
+            modifiers={modifiers}
+            showTrigger={[]}
+            hideTrigger={[]}
+            content={
+              <DialogContentContainer>
+                <ExampleContent />
+              </DialogContentContainer>
+            }
+          >
+            <Button
+              onFocus={onFocusButton}
+              onBlur={onFocusButton}
+              kind={Button.kinds.SECONDARY}
+              active={focusButtonActive}
+            >
+              On focus
+            </Button>
+          </Dialog>
```

Comment:
Wasn't able to see this dialog after keyboard focus

Reply:
fixed